File: Java/4-Array-based-Programs/51-largest-element-in-an-array.java
class Main {
    public static void main(String[] args) {
        int[] arr = {25, 89, 47, 75, 14};
        int large = arr[0];

        for(int i=1; i<arr.length; i++) {
            if (arr[i] > large) {
                large = arr[i];
            }
        }
        
        System.out.println("Largest element is: " + large);
    }
}


// Using built-in Method:
// import java.util.Arrays;
// int large = Arrays.stream(arr).max().getAsInt();

File: Python/4-Array-based-Programs/51-largest-element-in-an-array.py
arr = [10, 5, 35, 21, 18]
large = arr[0]

for i in range(len(arr)):
    if arr[i]>large:
        large = arr[i]
    
print("Largest element is:", large)


# Using built-in method
# print(max(arr))